src/styles/_overrides.scss

```diff
@@ -213,6 +213,26 @@ $grid-float-breakpoint: $screen-sm-min !default;
   }
 }
 
+.pf-c-page__header {
+  background-image: none;
+  background-color: unset;
+}
+
+.pf-c-background-image::before {
+  filter: unset;
+}
+
+.pf-c-login,
+.login-pf {
+  background-image: none;
+  background-color: #030303;
+}
+
+.pf-c-about-modal-box__hero {
+  background-image: none !important;
+  background-color: transparent;
```

Comment:
I prefer remove the background in component or pass a className to override the style instead use !important

Reply:
Placing styles here is what I understood from your other comment: https://github.com/kiali/kiali-ui/pull/1217/files#r288317015. I probably misunderstood :)

The About component does not have a background image set. It is PF4 styles which are setting that image by default.

I'm trying with typestyle. I still need to do:

```js
const aboutBoxStyle = style({
  $nest: {
    '& .pf-c-about-modal-box__hero': {
      backgroundImage: 'unset !important',
      backgroundColor: 'transparent'
    }
  }
});
```
PF styles have the `!important` flag, so even when using a className the `!important` flag is needed to prevent the PF4 styles taking precedence. So, I'm not sure if it is any better.

Reply:
Oh I saw that we can change the background in a newest release of PF4 we are in 3.2 :( ok so go ahead , after GA we can make changes